translations/default/AdminCatalogFeature.xlf

```diff
@@ -2808,29 +3088,112 @@
   </file>
   <file original="src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
+      <trans-unit id="76e2082ba7b0d7b16661f4996549a675">
+        <source>Save list</source>
+        <target>Save list</target>
+        <note>Line: 61</note>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list_header.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="b318944bdc95134d6284f7e54040970a">
+        <source>Edit %combinations_number% combinations</source>
+        <target>Edit %combinations_number% combinations</target>
+        <note>Line: 46</note>
+      </trans-unit>
+      <trans-unit id="768678dd6637e9466e187158e6ddfbf5">
+        <source>Apply to %combinations_number% combinations</source>
+        <target>Apply to %combinations_number% combinations</target>
+        <note>Line: 43</note>
+      </trans-unit>
+      <trans-unit id="709f5d9d0fb0fd399e19b30ec1e7ac2f">
+        <source>Delete %combinations_number% combinations</source>
+        <target>Delete %combinations_number% combinations</target>
+        <note>Line: 66</note>
+      </trans-unit>
+      <trans-unit id="0a93bc6a803478fc23cdef1d0947bf76">
+        <source>Are you sure you want to delete selected combinations?</source>
```

Comment:
Looks like it should be in Admin.Catalog.Notification

Same for line 3121 (Deleting %total...)
and line 3126 (Deleting %done...)